src/sentry/models/issueviews.py

```diff
@@ -0,0 +1,40 @@
+from django.db import models
+from django.db.models import UniqueConstraint
+
+from sentry.backup.scopes import RelocationScope
+from sentry.db.models import region_silo_model
+from sentry.db.models.base import DefaultFieldsModel
+from sentry.db.models.fields.foreignkey import FlexibleForeignKey
+from sentry.models.savedsearch import SortOptions
+
+
+@region_silo_model
+class IssueViews(DefaultFieldsModel):
+    """
+    A model for a user's view of the issue stream
+    """
+
+    __relocation_scope__ = RelocationScope.Organization
```

Comment:
I'm not 100% confident this is correct, but this is what was in savedsearches and is what I thought made sense given the description in `scopes.py`. Please correct me if this is wrong!

Reply:
This looks correct to me. What this does is include the model in self-hosted -> saas relocations, and in region -> region relocations when that is built.